aries-site/src/pages/guidelines/index.js

```diff
@@ -1,10 +1,22 @@
 import React from 'react';
-import { PageLayout, NavPage } from '../../layouts';
-import { DescriptiveHeader } from '../../components/headings';
+import { Box } from 'grommet';
+import { Button } from 'aries-core';
+
+import {
+  ButtonRow,
+  NavPage,
+  PageLayout,
+  ContentSection,
+  Subsection,
+} from '../../layouts';
+import { DescriptiveHeader, Meta, SubsectionText } from '../../components';
 import { structure } from '../../data';
 
 const title = 'Guidelines';
 const topic = structure.find(page => page.name === title);
+const description =
```

Comment:
Noticing this is also defined in structure -- is it possible for us to pull it from there instead?

Reply:
Discussed offline. Going to create seoDescription property in structure.js and use it to populated seo meta descriptions